src/browser/commands/types.ts

```diff
@@ -64,6 +64,21 @@ export interface AssertViewOpts extends Partial<AssertViewOptsConfig> {
      * @defaultValue `true`
      */
     disableAnimation?: boolean;
+    /**
+     * Ability to ignore small amount of different pixels while screenshots are still being considered as "same"
```

Comment:
Ability to ignore a small amount of different pixels to classify screenshots as being "identical"